src/ui/public/directives/__tests__/filter_invert.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+
+describe('FilterInvert directive', function () {
+  let $scope;
+  let $parentScope;
+  let $elem;
+
+  beforeEach(ngMock.module('kibana'));
+  beforeEach(ngMock.inject(function ($compile, $rootScope) {
+    $elem = angular.element('<img src="test.png" filter-invert="100%">');
+    $parentScope = $rootScope;
+    $compile($elem)($parentScope);
+    $elem.scope().$digest();
+    $scope = $elem.isolateScope();
+  }));
+
+  it('sets CSS filter:invert property in supported browsers', function () {
+    expect($elem.attr('style')).to.contain('filter: invert(100%)');
+  });
+});
```

Comment:
I'd like to see a test verifying the functionality for unsupported browsers as well.